cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/LedgerEvents/Gov/InfoAction.hs

```diff
@@ -7,6 +8,10 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 
+#if __GLASGOW_HASKELL__ >= 908
+{-# OPTIONS_GHC -Wno-x-partial #-}
```

Comment:
Which function(s) are partial?

Reply:
In the governance tests we're using `!!`, `head` and similar a lot, so that's where it comes from. I guess it's a good reminder to refactor this later to not use partial functions.